src/helpers/contextCreatorHelpers/index.test.js

```diff
@@ -74,6 +76,75 @@ describe('contextCreatorErrorHelper', () => {
     });
   });
 
+  describe('addErrorsAndDataToManifestKeyValuePair', () => {
+    const manifestKeyValueQuestions = {
+      title: 'Title of page',
+      questions: {
+        question1: {
+          id: 'question1',
+          mainAdvice: 'question 1 main advice',
+        },
+        question2: {
+          id: 'question2',
+          mainAdvice: 'question 1 main advice',
+        },
+        question3: {
+          id: 'question3',
+          mainAdvice: 'question 3 main advice',
+        },
+      },
+      errorMessages: {
+        Field1Required: 'Field 1 is required',
+        Field1TooLong: 'Field 1 is too long',
+        Field2Required: 'Field 2 is required',
+      },
+    };
+    const formattedErrors = formatErrors({ manifest: manifestKeyValueQuestions, errors });
+    it('should return the contents of the manifest', () => {
+      const manifestWithErrors = addErrorsAndDataToManifestKeyValuePair(
```

Comment:
so this function will change the manifest in key value pairs into array of questions? 
We just got to be bit cautious here. If in the normal scenario the template is expecting the questions as key value pair and then after we get validation errors it changes this to an array we are not going to have the same type. 

Do you think that this should also return the manifest in the same type ie key value pairs?